src/content/en/2024/sustainability.md

```diff
@@ -320,7 +320,7 @@ The not-so-nice surprise here is that most of the requests, around 70 in the 90t
 {{ figure_markup(
   image="kilobytes-by-percentile-by-type-mobile-.png",
   caption="Kilobytes by percentile by type (mobile)",
-  description="TODO",
+  description="A column chart showing that on mobile devices, on the 90th percentile we find 145.4 KB of HTML, around 1,732.17 KB of JavaScript, 260.4 KB of CSS, 4,436.1 KB of images and 399.33 KB of fonts. On the 75th percentile it goes down at 70.03 KB of HTML, 1,102.63 KB of JavaScript, 145.59 KB of CSS, 1,807.67 KB of images and 221.51 KB of fonts. On the 50th percentile we found 31.64 KB of HTML, 569.66 KB of JavaScript, 74.99 KB of CSS, 623.8 KB of images and 109.06 KB of fonts. On the 25th percentile there is 13.12 KB of HTML, 244.33 KB of JavaScript, 32.26 KB of CSS, 179.76 KB of images and 39.46 KB of fonts. On the 10th percentile there is 5.87 KB of HTML, 88.81 KB of JavaScript, 7.55 KB of CSS, 42.71 KB of images and 0 KB of fonts.",
```

Comment:
Here the decimals at least makes sense, but they don't add much value IMO. But again, since we have them in the chart I'm keeping them in the description

Reply:
In the [Figures Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) re recommend not having too many decimal places as it’s often more distracting and difficult to read than fewer ones:

> Avoid using too many decimal places. In general, when deciding how many decimal places to use data should have two "real figures". So 56% is good, but 56.45% is overly detailed. For some topics (or stats within a chapter) with more niche numbers, decimal places may be required (e.g. 4.5% or 0.00012%).